x/token/keeper/send.go

```diff
@@ -21,10 +21,11 @@ func (k Keeper) Send(ctx sdk.Context, contractID string, from, to sdk.AccAddress
 
 func (k Keeper) AuthorizeOperator(ctx sdk.Context, contractID string, holder, operator sdk.AccAddress) error {
 	if _, err := k.GetClass(ctx, contractID); err != nil {
-		return err
+		panic(err)
```

Comment:
Because the caller must ensure the condition is true. I may remove the entire check logic if it looks strange.

Reply:
I thought that caller may be able to input contractID incorrectly sometimes because of human error. And the method such as `TransferFT` returns error instead of panic. So I guess every msg which consume `contractID` should have same flow for consistency.

Reply:
Generally, you are correct. However, we must always check the existence when we get `contractID` as a parameter. So, the focus should be on "what is the interface". Unfortunately, the function is, as of now, public.
The correct way to fix this is, introducing `internal` and hide the private functions. However, this module is deprecated one, so we won't spend any time refactoring it.
FYI: The function has been made public for black-box testing.